pkg/webui/components/form/field/tooltip.js

```diff
@@ -0,0 +1,97 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import Tooltip from '@ttn-lw/components/tooltip'
+import Icon from '@ttn-lw/components/icon'
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import fieldDescriptions from '@ttn-lw/lib/field-description-messages'
+
+import style from './field.styl'
+
+const m = defineMessages({
+  descriptionTitle: 'What is it?',
+  locationTitle: 'Where can I find it?',
+  absenceTitle: "What if I can't find it?",
+  viewGlossaryPage: 'View glossary page',
+})
+
+const Content = props => {
+  const { tooltipInfo, glossaryId } = props
+  const { description, location, absence } = tooltipInfo
+
+  const hasLocation = Boolean(location)
+  const hasAbsence = Boolean(absence)
+
+  return (
+    <div>
+      <Message className={style.tooltipTitle} content={m.descriptionTitle} component="h4" />
+      <Message className={style.tooltipDescription} content={description} component="p" />
+      {hasLocation && (
+        <>
+          <Message className={style.tooltipTitle} content={m.locationTitle} component="h4" />
+          <Message className={style.tooltipDescription} content={location} component="p" />
+        </>
+      )}
+      {hasAbsence && (
+        <>
+          <Message className={style.tooltipTitle} content={m.absenceTitle} component="h4" />
+          <Message className={style.tooltipDescription} content={absence} component="p" />
+        </>
+      )}
+      <Link.GlossaryLink term={m.viewGlossaryPage} glossaryId={glossaryId} />
```

Comment:
Let's additionally make it possible to link to arbitrary pages in the documentation. This will be useful if we have a dedicated page somewhere in the docs about a specific field or topic.

Reply:
Do we already have pages to link? Otherwise I would prefer adding this later.

Do you refer to the `Additional information` link?
<img width="440" alt="Screenshot 2021-04-20 at 08 06 13" src="https://user-images.githubusercontent.com/16374166/115340383-39713000-a1af-11eb-8b00-d4ba643443d5.png">

Reply:
Yes it would be that. We definitely have pages to link. E.g. for Frequency Plans as in your picture. It would be [https://www.thethingsnetwork.org/docs/lorawan/frequency-plans](https://www.thethingsnetwork.org/docs/lorawan/frequency-plans)

So it should be possible to use both `<Link.Doc />` as well as `<Link.Anchor />`. Maybe it makes more sense to use the `children` prop for this.